### ISSUE TITLE
#4 Program Accuracy + Macro-F1 + confusion matrix

### ISSUE THREAD

**Issue by sigurdurhaukur** [ponderingpenguins/natural-language-processing#5]
Use something like this (from the tutorial)

```py
from sklearn.metrics import classification_report, confusion_matrix, ConfusionMatrixDisplay
import matplotlib.pyplot as plt

# Make predictions
y_pred = model.predict(X_test)

print(classification_report(y_test, y_pred, target_names=newsgroups_train.target_names))

cm = confusion_matrix(y_test, y_pred)
disp = ConfusionMatrixDisplay(confusion_matrix=cm, display_labels=newsgroups_train.target_names)
disp.plot(xticks_rotation='vertical')
plt.title("Confusion Matrix: TF-IDF + Logistic Regression")
plt.show()
```
